cmd/scollector/collectors/collectors.go

```diff
@@ -62,6 +62,21 @@ const (
 )
 
 const (
+	bgpPeerState               = "bgp.peer.state"
```

Comment:
Will do - was following the conventions from snmp_ifaces which I assumed were an attempt to enable easier localization of these strings?